.github/workflows/ci.yml

```diff
@@ -22,9 +22,8 @@ jobs:
 
     runs-on: ${{ matrix.os || 'ubuntu-20.04' }}
 
-    # By default, the name of the build is the language used and SWIG options, but matrix entries
-    # can define the additional "desc" field with any additional information to include in the name.
-    name: ${{ matrix.SWIGLANG || 'none' }}${{ matrix.PY3 }} ${{ matrix.ENGINE}} ${{ matrix.VER }} ${{ matrix.SWIG_FEATURES }} ${{ (matrix.compiler || 'gcc') }}${{ matrix.GCC }} ${{ matrix.CPPSTD }} ${{ matrix.CSTD }} ${{ matrix.desc }} ${{ matrix.continue-on-error && '(can fail)' }}
+    # By default, the name of the build is the language used and SWIG options.
+    name: ${{ matrix.SWIGLANG || 'none' }}${{ matrix.PY3 }} ${{ matrix.ENGINE}} ${{ matrix.VER }} ${{ matrix.SWIG_FEATURES }} ${{ (matrix.compiler || 'gcc') }}${{ matrix.GCC }} ${{ matrix.CPPSTD }} ${{ matrix.CSTD }} ${{ matrix.continue-on-error && '(can fail)' }}
```

Comment:
This dropping of `desc` doesn't seem related to adding a C backend.  It also seem a useful feature, though TBH I've barely looked at the new CI stuff.

Reply:
Thanks, this looks like a bad merge, I'll fix this.

Reply:
Ah, sorry, no, I remember now: I've added `desc` to use it for anything not covered by the other variables in the build descriptions but @wsfulton removed all uses of it in 04a6ad3bb582c15d7dc13c86b9657038adeb07ce and e74b9830bbb0a3f32d044e56b671b7685cd253bf, so I've decided to remove it here too. We can keep it, of course, but to me it seems confusing to have a never used variable.